js/src/attribute-mapping/index.scss

```diff
@@ -0,0 +1,17 @@
+.gla-settings-attribute-mapping {
+
+	&__notice {
+		background: $gla-color-light-blue;
+		margin: 0;
+		margin-bottom: $grid-unit-20;
+
+		&.is-dismissible {
+			padding-right: 10px;
+		}
+	}
+
+	&__table-footer {
+		justify-content: flex-start;
```

Comment:
💅 Looks like this style could be omitted since it's already specified at:
https://github.com/woocommerce/google-listings-and-ads/blob/5cbc2f804bd11cf5afea7a06b3de2fc840d4be7b/js/src/attribute-mapping/attribute-mapping-table.js#L60